aws-datastore/src/test/java/com/amplifyframework/datastore/syncengine/MutationProcessorTest.java

```diff
@@ -154,11 +153,7 @@ public void canDrainMutationOutbox() throws DataStoreException {
         mutationProcessor.startDrainingMutationOutbox();
 
         // Assert: the event was published
-        List<HubEvent<?>> events = accumulator.await();
-        assertEquals(1, events.size());
-        @SuppressWarnings("unchecked")
-        PendingMutation<BlogOwner> mutation = (PendingMutation<BlogOwner>) events.get(0).getData();
-        assertEquals(createTony, mutation);
```

Comment:
wouldn't this have been a stronger assertion?

Reply:
It was duplicative logic. The `isProcessed` filter already checks for the model type and ID. So if the accumulator even gets an event, it means we did find this match, already.

In other words, this assertion is sort of just testing the `isProcessed` filter again.